language/tools/move-unit-test/tests/test_sources/native_signer_creation.exp

```diff
@@ -2,6 +2,12 @@ Running Move unit tests
+0x1::M::test_exists
+Output: Ok(ChangeSet { accounts: {00000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 01000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 02000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 03000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 04000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 05000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 06000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 07000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 08000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }, 09000000000000000000000000000000: AccountChangeSet { modules: {}, resources: {StructTag { address: 00000000000000000000000000000001, module: Identifier("M"), name: Identifier("A"), type_params: [] }: New([0])} }} })
```

Comment:
This really feels like a bit of a hack, and I don't find the output here really readable. 
Could we at least change the format here to be something human readable rather than the debug dump?

Reply:
More on this in #483